### DISCUSSION TITLE
4.1.1 ok, but 4.1.2 cant find pom file.

### DISCUSSION THREAD

**Discussion by xlinx** [micycle1/processing-core-4#5] [General]
Dear Micycle.
thx for this marven, i use it as well 4.1.1, but the 4.1.2 can not find jogl-all-2.4.0-rc-20230123.pom file on NGIX server. error trace as follow:

> Could not resolve all files for configuration ':runtimeClasspath'.
   > Could not find org.jogamp.jogl:jogl-all:2.4.0-rc-20230123.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jogamp/jogl/jogl-all/2.4.0-rc-20230123/jogl-all-2.4.0-rc-20230123.pom
       - https://jitpack.io/org/jogamp/jogl/jogl-all/2.4.0-rc-20230123/jogl-all-2.4.0-rc-20230123.pom
     Required by:
         project : > com.github.micycle1:processing-core-4:4.1.2
   > Could not find org.jogamp.gluegen:gluegen-rt-main:2.4.0-rc-20230123.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jogamp/gluegen/gluegen-rt-main/2.4.0-rc-20230123/gluegen-rt-main-2.4.0-rc-20230123.pom
       - https://jitpack.io/org/jogamp/gluegen/gluegen-rt-main/2.4.0-rc-20230123/gluegen-rt-main-2.4.0-rc-20230123.pom
     Required by:
         project : > com.github.micycle1:processing-core-4:4.1.2

**Comment by micycle1** [comment]
Maven should try to get these two artefacts from https://jogamp.org/deployment/maven/ (which has been added as a repository in the pom), but for some reason yours is not picking it up.
